config/jobs/kubernetes/sig-node/crio.yaml

```diff
@@ -288,3 +288,34 @@ periodics:
     testgrid-tab-name: ci-crio-cgroupv1-node-e2e-hugepages
     testgrid-alert-email: kubernetes-sig-node-test-failures@googlegroups.com
     description: "Executes hugepages e2e tests"
+- name: ci-crio-cgroupv1-node-e2e-performance
+ interval: 12h
+ labels:
+   preset-service-account: "true"
+   preset-k8s-ssh: "true"
+ spec:
+   containers:
+     - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20211217-ea95cec1d4-master
+       args:
+         - --repo=k8s.io/kubernetes=master
+         - --timeout=90
+         - --root=/go/src
+         - --scenario=kubernetes_e2e
+         - --
+         - --deployment=node
+         - --gcp-project-type=node-e2e-project
+         - --gcp-zone=us-west1-b
+         - --node-args=--image-config-file=/workspace/test-infra/jobs/e2e_node/perf-image-config.yaml
```

Comment:
I believe this file needs to be changed but have no idea why and how?

Reply:
Yes, this is an ubuntu image. It might be easier to first migrate this to containerd on Ubuntu before trying CRI-O on fedora. See https://github.com/kubernetes/test-infra/blob/master/jobs/e2e_node/perf-image-config.yaml for the image config

Reply:
> It might be easier to first migrate this to containerd on Ubuntu before trying CRI-O on fedora.

opened a separate PR for this: https://github.com/kubernetes/test-infra/pull/25254

@ehashman - for the CRI-O version of this job (this pr) which would be the best fedora image we should refer to, from this list: https://github.com/kubernetes/test-infra/tree/master/jobs/e2e_node/crio/latest? thanks

Reply:
> opened a separate PR for this: https://github.com/kubernetes/test-infra/pull/25254

@Namanl2001 this PR didn't work: https://testgrid.k8s.io/sig-node-containerd#node-kubelet-containerd-performance-test can you please take a look?